docs/discussion/maintenance-schedules.rst

```diff
@@ -0,0 +1,78 @@
+Maintenance Schedules
+===========================================================
+
+This is a record of the intended maintenance schedules for various releases
+and dependency support.
+
+Python Version Support
+-----------------------------------------------------------
+
+PursuedPyBear projects will support at least the latest version of
+`pypy <https://www.pypy.org/download.html>`_ and the latest two versions of
+`C Python <https://www.python.org/downloads/>`_
+
+We tend to start testing new versions of cPython when they reach the release
+candidate stage, but don't guarantee compatibility until at least one ppb
+release after the latest Python release.
+
+We drop older versions of Python when we begin using newer features. This
+means the oldest version of Python ppb will support may be older than the
+previous minor version of Python. This is especially true because of our
+support for pypy.
+
+Release Schedules
+------------------------------------------------------------
+
+PursuedPyBear targets four releases a year, based on the solstices and
+equinoxes:
+
+* Around the Northward equinox, which is about March 20.
```

Comment:
is the capitalization really Solstice and equinox?

Reply:
. . . Honestly, I am unsure.

Solstice seems to be inconsistent (and the capitalization may be because witchy), but equinox is almost always lowercase.